config.yml

```diff
@@ -0,0 +1,35 @@
+defaults:
+  hostname:           https://events.taskcluster.net
+  port:               undefined
+  forceSSL:           false
+  trustProxy:         false
+  influx:
+    maxDelay:         5*60
+    maxPendingPoints: 250
+  events:
+    statsComponent:   events
+
+production: # profile 'production'
+  hostname:           !env HOSTNAME
+  port:               !env:number PORT
+  pulse:
+    username:         !env PULSE_USERNAME
+    password:         !env PULSE_PASSWORD
+  taskcluster:
+    credentials:
+      accessToken:    !env TASKCLUSTER_CREDENTIALS_ACCESSTOKEN
+      clientId:       !env TASKCLUSTER_CREDENTIALS_CLIENTID
```

Comment:
These should be: TASKCLUSTER_CLIENT_ID and TASKCLUSTER_ACCESS_TOKEN like everywhere else..